### ISSUE TITLE
Can't use my options

### ISSUE THREAD

**Issue by cagedThespian** [illuminatedwax/pesterchum#155]
My options wont open. I will click on the options button and it wont open, i have tried many times and still nothing. A friend of mine told me to just register my chum handle but that didnt work.

**Comment by arcaneAgilmente** [comment]
I'll be assuming you're running the portable (Or something based off of it)
build right? If that's the case make sure you have a default profile set
before starting the program. That'll allow the options menu to appear.

**Comment by cagedThespian** [comment]
Yes i am using the portable version because i am unable to download to my computer and how do i do that? I am still new to pesterchum so i dont know much about how to do a lot of it.

**Comment by arcaneAgilmente** [comment]
If you have the program open, close it by navigating to CLIENT => EXIT .
Reopen the program. When the select Chumhandle screen pops up, Select the
handle you'd like as the programs default from the dropdown box. Then tick
the box that says "use this profile as the default". Click okay.
If you're using the portable build (or something using it as it's base),
you might have to restart the program once more after doing this before the
options menu appears. Once it does, it should appear from then on and the
program will work as normal.

**Comment by zeronobody0** [comment]
I'm having the same problem on th PC build. Exept it crashes.
